documentation/sphinx/conf.py

```diff
@@ -143,7 +143,7 @@
 # Add any paths that contain custom static files (such as style sheets) here,
 # relative to this directory. They are copied after the builtin static files,
 # so a file named "default.css" will overwrite the builtin "default.css".
-html_static_path = [sys.prefix + '/_static']
+#html_static_path = [sys.prefix + '/_static']
```

Comment:
Are we seeing problems from not including the `_static` thing? I guess my concern is that someone might try to add `_static` files here (if that's a standard sphinx thing, which I think it is) and be surprised when it doesn't do anything (until they eventually find that we've commented out this line).